tools/odbc/include/statement_functions.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "duckdb_odbc.hpp"
+
+namespace duckdb {
+
+static SQLRETURN PrepareStmt(SQLHSTMT StatementHandle, SQLCHAR *StatementText, SQLINTEGER TextLength) {
```

Comment:
Why is this function body in a header?

Reply:
Nothing special. I will move the implementation to a source file.